src/commonMain/kotlin/org/kbson/ext/CurrentTime.kt

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2008-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.kbson.ext
+
+expect fun getCurrentTimeInMillis(): Long
```

Comment:
These seems to be internal only. In the Realm Kotlin SDK we moved all of these to a new internal package like `org.kbson.internal.ext`. 

The reason for that is that while we could just mark them as `internal`, they would still turn into public methods when called from Java code, so having an explicit internal package makes it more explicit what this is.

Reply:
Ack